### ISSUE TITLE
WhatsApp Web message dialog won't show in Google Chrome

### ISSUE THREAD

**Issue by cnekmp** [qtile/qtile#2586]
When on Qtile, WhatsApp Web message dialog won't show anything on Google Chrome when clicked. While there is no issues on Firefox (Qtile) or when I log back to other DE/WM the issue won't appear also. There is no errors in qtile/xsession/chrome logs.

Qtile version: qtile 0.17.0-1
Chrome version: google-chrome 91.0.4472.114-1
OS: ArchLinux

Video of issue:
https://user-images.githubusercontent.com/35117097/123639515-5706e980-d831-11eb-8198-9c67a0deb4ce.mp4

**Comment by elParaguayo** [comment]
Can you try this: https://github.com/qtile/qtile/issues/2404#issuecomment-841300614

If possible, could you also try the latest git version as a fix for a similar issue has been merged but it's not in 0.17.0

**Comment by ramnes** [comment]
Works for me, I think it has been fixed with #2506. Can you try with the latest commit?

**Comment by cnekmp** [comment]
> Can you try this: [#2404 (comment)](https://github.com/qtile/qtile/issues/2404#issuecomment-841300614)
> 
> If possible, could you also try the latest git version as a fix for a similar issue has been merged but it's not in 0.17.0

1. [#2404 (comment)](https://github.com/qtile/qtile/issues/2404#issuecomment-841300614) solved issue as a remedy.
2. Package `aur/qtile-git 3444.458c4875-2` failed to start even. However no qtile failure logs observed during startup.

**Comment by elParaguayo** [comment]
That's not the actual version number, it's just the version of the PKGBUILD in the AUR. Can you do a `pacman -Qi qtile-git` to see what version it is.

It should definitely start so something's gone wrong there.

**Comment by cnekmp** [comment]
> That's not the actual version number, it's just the version of the PKGBUILD in the AUR. Can you do a `pacman -Qi qtile-git` to see what version it is.
> 
> It should definitely start so something's gone wrong there.

Sure.

[x@unknown ~]$  pacman -Qi qtile-git
Name            : qtile-git
Version         : 4183.d5425eeb-1
Description     : A full-featured, pure-Python tiling window manager. (git version)
Architecture    : any
URL             : http://www.qtile.org
Licenses        : MIT
Groups          : None
Provides        : qtile
Depends On      : python  pango  python-xcffib  python-cairocffi
Optional Deps   : python-setproctitle: change the process name to qtile [installed]
Required By     : None
Optional For    : None
Conflicts With  : qtile
Replaces        : None
Installed Size  : 2.22 MiB
Packager        : Unknown Packager
Build Date      : Tue 29 Jun 2021 12:23:38 AM +04
Install Date    : Tue 29 Jun 2021 01:16:55 AM +04
Install Reason  : Explicitly installed
Install Script  : Yes
Validated By    : None

**Comment by elParaguayo** [comment]
Great. That's definitely the latest.

How are you starting qtile? If the logs aren't being populated then there may be some errors being displayed elsewhere.

**Comment by cnekmp** [comment]
> Great. That's definitely the latest.
> 
> How are you starting qtile? If the logs aren't being populated then there may be some errors being displayed elsewhere.

Ok. I've solved the `qtile-git` package starting issue. I was starting it via `.xinitrc` value `exec qtile`. Turns out, the latest `qtile-git` version requires additional `start` parameter in xinitrc config. Now it works normally with following config:

```
[farid@unknown ~]$ cat .xinitrc | grep exec
exec qtile start
[farid@unknown ~]$ qtile -v
0.17.1.dev354+gd5425eeb
```

Final result:
1.  [#2404 (comment)](https://github.com/qtile/qtile/issues/2404#issuecomment-841300614) solved issue.
2. Using latest Qtile build without browser startup key modification solved issue too. Just keep in mind start parameter issue that I've mentioned above.

If it's ok with such startup parameter, this case could be closed. Thanks a lot.
